cypress/integration/e2e/header-control-menu/return-to-now-button.spec.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+describe("'Come back to now' button", () => {
+  context("when schedule is loaded", () => {
+    before(() => {
+      cy.loadScheduleToMonth();
+      cy.get("[data-cy=switch-next-month]").click();
+    });
+
+    context("when month is switched to next", () => {
+      it("should get you back to current month after clicking 'return to now' button", () => {
+        cy.get("[data-cy=return-to-now]").click();
+        cy.get("[data-cy=month-name]").contains("Listopad 2020");
```

Comment:
Dodaj dodatkowy check, to co chcemy żeby ten button zrobił to żeby wrócił do grafiku aktualnego miesiąca, więc dobrze sprawdzić też tabelkę, może wystarczy tylko sprawdzenie dni miesiąca (polecam użyć do tego snapshot np.`cy.get('[data-cy="team0ShiftsTable"] [data-cy="1Row"] div[data-cy*="cell"]')
          .then(($cell) => $cell.map((i, el) => Cypress.$(el).text()).get())
          .snapshot();` (cypress/integration/e2e/table/copy-schedule.spec.ts) ale jak masz jakiś inny pomysł też spoko